docs/src/HomePage.js

```diff
@@ -20,14 +21,22 @@ export default class HomePage extends React.Component{
         </main>
 
         <Grid>
+          <Alert bsStyle='warning'>
+            <p><Glyphicon glyph='bullhorn' /> The project needs contributors. <
+              Glyphicon glyph='user' /> <
+              Glyphicon glyph='user' /> <
+              Glyphicon glyph='user' />
+            </p>
+            <p>Check out the <a href="https://github.com/react-bootstrap/react-bootstrap/wiki#100-roadmap">1.0.0 Roadmap</a> and <a href="https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md">Contributing
+            Guidelines</a> to see where you can help out.</p>
+            <p>The good place to start is <a target='_blank'
+            href="https://github.com/react-bootstrap/react-bootstrap/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22">issues
+            </a> with a <Label bsStyle='success'>help-wanted</Label> label.</p>
+            <p>If anyone is interested and would like to help out we are very open to pull requests. <Glyphicon glyph='thumbs-up' /></p>
+          </Alert>
           <Alert bsStyle='danger'>
-            <Glyphicon glyph='warning-sign' /> The project is under active
-            development, and APIs will change. Check out the <a
-            href='https://github.com/react-bootstrap/react-bootstrap/wiki#100-roadmap'>1.0.0
-            Roadmap</a> and <a
-            href='https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md'>Contributing
-            Guidelines</a> to see where you can help out. Prior to the 1.0.0 release, breaking changes should
-            result in a Minor version bump.
+            <p><Glyphicon glyph='warning-sign' /> The project is under active development, and APIs will change. </p>
+            <p><Glyphicon glyph='bullhorn' /> Prior to the 1.0.0 release, breaking changes should result in a Minor version bump.</p>
```

Comment:
I'd drop the bullhorn here and join these two lines, mainly so it will look a little less cluttered.